Client/src/Pages/Infrastructure/Details/index.jsx

```diff
@@ -183,11 +186,14 @@ const InfrastructureDetails = () => {
 	useEffect(() => {
 		const fetchData = async () => {
 			try {
-				const response = await axios.get("http://localhost:5000/api/v1/dummy-data", {
-					headers: {
-						"Content-Type": "application/json",
-						"Cache-Control": "no-cache",
-					},
+				const response = await networkService.getStatsByMonitorId({
+					authToken: authToken,
+					monitorId: monitorId,
+					sortOrder: null,
+					limit: null,
+					dateRange: dateRange,
+					numToDisplay: 50,
+					normalize: true,
```

Comment:
We don't need to normalize results here, normalizing the data is used when we are displaying response time in a chart.  Response time has an unbounded range so we need to normalize the data to have a reasonable visual representation.

Since we're not diplaying response time here this can safely be set to false.